app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,30 @@
+class PetApplicationsController < ApplicationController
```

Comment:
Nice use of the joins controller to adhere to RESTful routing!! Using `create` for adding a pet to an application and `update` to setting the status of the pet_application is the most RESTful choice